ParkPlanner/Facilities.cs

```diff
@@ -57,19 +57,19 @@ public partial class SearchParameters
     public partial class Recdatum
     {
         [JsonProperty("ACTIVITY", NullValueHandling = NullValueHandling.Ignore)]
-        public List<object> Activity { get; set; }
```

Comment:
1. Use IEnumerable<T> Instead of List<T> When Possible:
     If the order of the elements is not essential, consider using IEnumerable<T> instead of List<T> in property declarations. 
     This provides more flexibility when it comes to changing the implementation later.

Reply:
I could go either way on this.  IEnumerable is a less-specific type than list.  Since this is a QuickType-generated class, I'd leave it as it were, as a List.